apps/demo/partials/header.hbs

```diff
@@ -1,20 +1,136 @@
+      {{#if @is_development}}
+        <li class="HeaderNavItem">
+          <a href="https://localhost:4443" class="HeaderLink">Web Twig</a>
+        </li>
+      {{/if}}
+      <li class="HeaderNavItem">
+        <a
+          href="/packages/form-validations/"
+          class="HeaderLink {{# if (eq package 'Form Validations')}}HeaderLink--current{{/if}}"
+          {{# if (eq package 'Form Validations')}}aria-current="page"{{/if}}
+        >
+          Form Validations
+        </a>
+      </li>
+    </ul>
+    <!-- HeaderNav: end -->
+  </nav>
+  <!-- HeaderDesktopActions: end -->
+
+</header>
+
+<!-- HeaderDialog: start -->
+<dialog id="off-canvas-navigation" class="HeaderDialog" aria-label="Menu">
+  <div class="HeaderDialog__panel">
+    <div class="HeaderDialog__content">
 
-    <div class='d-grid d-tablet-block'>
-      <a
-        href='https://github.com/lmc-eu/spirit-design-system'
-        class='Button Button--tertiary Button--large mb-400 mb-tablet-0 mr-tablet-400'
+      <!-- HeaderDialogCloseButton: start -->
+      <button
+        type="button"
+        class="HeaderDialogCloseButton"
+        data-spirit-dismiss="off-canvas-navigation"
+        data-spirit-target="#off-canvas-navigation"
+        aria-controls="off-canvas-navigation"
+        aria-expanded="false"
       >
-        GitHub Project
-      </a>
-      <a href='https://spirit.supernova-docs.io/spirit/' class='Button Button--tertiary Button--large'>
-        Spirit Docs →
-      </a>
-    </div>
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/assets/icons/svg/sprite.svg#close" />
+        </svg>
+        <span class="accessibility-hidden">Close</span>
+      </button>
+      <!-- HeaderDialogCloseButton: end -->
 
+      <!-- HeaderDialogActions: start -->
+      <nav class="HeaderDialogActions HeaderDialogActions--primary" aria-label="Main navigation">
+
+        <!-- HeaderDialogNav: start -->
+        <ul class="HeaderDialogNav">
+          <li class="HeaderDialogNavItem">
+            <a
+              href="/packages/web/"
+              class="HeaderDialogLink {{# if (eq package 'Web')}}HeaderDialogLink--current{{/if}}"
+              {{# if (eq package 'Web')}}aria-current="page"{{/if}}
+            >
+              Web
+            </a>
+          </li>
+          <li class="HeaderDialogNavItem">
+            <a
+              href="/packages/web-react/"
+              class="HeaderDialogLink {{# if (eq package 'Web React')}}HeaderDialogLink--current{{/if}}"
+              {{# if (eq package 'Web React')}}aria-current="page"{{/if}}
+            >
+              Web React
+            </a>
+          </li>
+          {{!-- Hide link to the Twig implementation in production. Twig demo app only runs on localhost, we don't deploy it anywhere. --}}
+          {{#if @is_development}}
+            <li class="HeaderDialogNavItem">
+              <a href="https://localhost:4443" class="HeaderDialogLink">Web Twig</a>
```

Comment:
I would love to see this imported from the common package. Maybe we can create an issue to let someone do it :)

Reply:
Unfortunately, the Vite config refuses to load any data from the `common` package. Maybe @literat will know, but it looks like a separate issue indeed.

```js
// apps/demo/config/vite/app.ts

import { SERVERS } from '@lmc-eu/spirit-common/constants/servers';

export default defineConfig({
  // …
  server: {
    hmr: {
      overlay: false,
    },
    ...SERVERS.DEVELOPMENT.web,
  },
});
```
```
failed to load config from /Users/adam.kudrna/Web/spirit-design-system/apps/demo/config/vite/app.ts
error when starting dev server:
TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /Users/adam.kudrna/Web/spirit-design-system/packages/common/constants/environments.ts
```

Reply:
Filed as https://jira.almacareer.tech/browse/DS-1506.